README.md

```diff
@@ -95,56 +87,20 @@ let device: Device = instance
     .unwrap();
 ```
 
-To not lose this lifetime single items can be "cast" to a slice of length _one_ with `std::slice::from_ref` while still taking advantage of `Deref`:
-
-```rust
-let queue_info = vk::DeviceQueueCreateInfo::builder()
-    .queue_family_index(queue_family_index)
-    .queue_priorities(&priorities);
-
-let device_create_info = vk::DeviceCreateInfo::builder()
-    .queue_create_infos(std::slice::from_ref(&queue_info))
-    ...;
-```
```

Comment:
Not sure if it's still useful to mention `std::slice::from_ref`? I'm unsure if `let x = [vk::...]` or `let x = &[vk::...]` make an extra stack allocation or if there are other cases where moving a singular Vulkan struct into an array of length-1 is not feasible (i.e. if you only have a borrow of a struct as function parameter, but the Vulkan API requires a slice).

Reply:
Note inline arrays can be used just fine if the structure doesn't need to outlive the enclosing statement, e.g. when constructed directly in a Vulkan function's argument list, which is my personal preferred pattern.

I think `slice::from_ref` is a bit niche and there's no harm in constructing intermediate arrays if you need to. It's not really meaningful to discuss whether there's additional stack allocation there, since LLVM will happily shuffle such straightforward things around into whatever form makes the most sense.

Reply:
Fair enough, I do think `let` bindings have their place in massive nested structs like the render passes listed above, but we at least have the full borrow-checking now to validate soundness and users can write it however they prefer :+1: 

Good, then I'll leave out the `from_ref` mention entirely.